templates/admin/payments-details.tpl.php

```diff
@@ -84,7 +84,7 @@
                                 <div class="wpbdp-admin-box-row">
                                     <label><?php esc_html_e( 'Gateway', 'business-directory-plugin' ); ?></label>
                                     <?php /* translators: Gateway: (Not yet set) */ ?>
-                                    <?php echo esc_html( $payment->gateway ? $payment->gateway : __( '(Not yet set)', 'business-directory-plugin' ) ); ?>
+                                    <?php echo esc_html( $payment->gateway ? sprintf( '%s %s', $payment->gateway, $payment->is_test ? __( '- Test Mode', 'business-directory-plugin' ) : '' )  : sprintf( '%s %s', __( '(Not yet set)', 'business-directory-plugin' ), $payment->is_test ? __( '- Test Mode', 'business-directory-plugin' ) : '' ) ); ?>
```

Comment:
There seems to be some duplication in here. Maybe we can simplify this? I think it would be fine to add the test mode part on the next line too rather than tacking it on here.

Reply:
Next line would be simpler. I just followed the example you had in the issue. I will update